po/ja.po

```diff
@@ -0,0 +1,561 @@
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:74
+msgid "Mark the active item as being a password"
+msgstr "アクティブな項目をパスワードとしてマークする"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:283
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:75
+msgid "Display the history"
+msgstr "履歴を表示する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:290
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:76
+msgid "Sync the clipboard to the primary selection"
+msgstr "クリップボードを一次選択に同期する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:297
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:77
+msgid "Sync the primary selection to the clipboard"
+msgstr "一次選択をクリップボードに同期する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:304
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:78
+msgid "Upload the active item to a pastebin service"
+msgstr "ペーストビン・サービスにアクティブな項目をアップデートする"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:319
+#: src/libgpaste/ui/gpaste-ui-window.c:311 src/client/gpaste-client.c:789
+msgid "Couldn't connect to GPaste daemon"
+msgstr "GPaste デーモンに接続できませんでした"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:334
+msgid "General behaviour"
+msgstr "一般行動"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:335
+msgid "History settings"
+msgstr "履歴設定"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:336
+msgid "Keyboard shortcuts"
+msgstr "キーボード・ショートカット"
+
+#. translators: This is the name of a multi-history management action
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:453
+#: src/libgpaste/ui/gpaste-ui-panel.c:297
+msgid "Switch to"
+msgstr "切り替える"
+
+#. translators: This is the name of a multi-history management action
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:458
+#: src/libgpaste/ui/gpaste-ui-delete-item.c:49
+#: src/libgpaste/ui/gpaste-ui-delete-history.c:25
+#: src/libgpaste/ui/gpaste-ui-delete-history.c:65
+msgid "Delete"
+msgstr "削除"
+
+#: src/libgpaste/ui/gpaste-ui-about.c:41
+msgid "About"
+msgstr "このプログラムについて"
+
+#: src/libgpaste/ui/gpaste-ui-backup-history.c:43
+#: src/libgpaste/ui/gpaste-ui-backup-history.c:140
+msgid "Backup"
+msgstr "バックアップ"
+
+#: src/libgpaste/ui/gpaste-ui-backup-history.c:44
+#: src/libgpaste/ui/gpaste-ui-edit-item.c:46
+#: src/libgpaste/ui/gpaste-ui-new-item.c:32 src/libgpaste/util/gpaste-util.c:30
+msgid "Cancel"
+msgstr "キャンセル"
+
+#: src/libgpaste/ui/gpaste-ui-backup-history.c:47
+msgid "Under which name do you want to backup this history?"
+msgstr "この履歴をどの名前でバックアップしますか？"
+
+#: src/libgpaste/ui/gpaste-ui-delete-history.c:25
+msgid "Are you sure you want to delete this history?"
+msgstr "この履歴を削除してもよろしいですか？"
+
+#: src/libgpaste/ui/gpaste-ui-edit-item.c:45
+#: src/libgpaste/ui/gpaste-ui-edit-item.c:118
+msgid "Edit"
+msgstr "編集"
+
+#. Translators: this is the translation for emptying the history
+#: src/libgpaste/ui/gpaste-ui-empty-history.c:65
+#: src/libgpaste/util/gpaste-util.c:410
+msgid "Empty"
+msgstr "履歴は空です"
```

Comment:
```suggestion
msgstr "空にする"
```

See the above comment for translators:

> Translators: this is the translation for emptying the history

Reply:
Thanks. I got a little confused with this one.